src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+# Frame V4 API FAQ
+
+The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to deliver.
+
+
+* How do I **authenticate**?
+
+    The first step in accessing the [Frame.io](http://frame.io/) API is getting authenticated. Create the user authentication credentials and then the access token. Adobe Identity Management Service (IMS) is Adobe’s OAuth 2.0-based identity management feature that supports authentication. There are two steps required for generating the access token:
+
+    * The user has to authorize the application before the token generation. The details for this can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#authorize-request).
+    * The next step is fetching the access token. The above step returns a **code** in the response body. This is used in this step. The exact endpoints to trigger and the parameters to use can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#fetching-access-tokens).
```

Comment:
I'm not sure we need to describe this process again here. My personal preference if we include this question would be just to link to the description in the Getting Started guide (or elsewhere). My rationale is that, just as with code, it's harder to keep multiple descriptions up-to-date as we make changes in the future.

Reply:
I took it out.